src/components/tooltip/NetworkMarker.js

```diff
@@ -69,6 +69,21 @@ function _getHumanReadableDataStatus(status: NetworkStatus): string {
   }
 }
 
+function _getHumanReadableHttpVersion(httpVersion: string): string {
+  switch (httpVersion) {
+    case 'h3':
+      return '3.0';
```

Comment:
nit: for both HTTP 2 and 3,  I think we don't need `.0`, it's commonly refered as HTTP 2 and HTTP 3, not HTTP 2.0 and HTTP 3.0 as far as I know.